src/filesystem/impls/appshell/node/node_modules/fsevents_win/fsevents_win.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014 - present Adobe Systems Incorporated. All rights reserved.
```

Comment:
This change, the base of your commit, seems to be a left-over of the intermediate work. Could you please rebase the commit on top of `adobe/master`?